src/pretix/control/views/global_settings.py

```diff
@@ -235,6 +235,14 @@ def _check_results(self, input):
 
         for entry_point in pkg_resources.iter_entry_points(group='pretix.plugin', name=None):
             license, url = self._get_license_for_pkg(entry_point.dist.key)
+            
+            if not license or not any(l in license for l in ('Apache', 'MIT', 'BSD', 'pretix Enterprise', 'GPL')):
+                res.append((
+                    'muted', 'warning',
+                    _('We found the plugin "{plugin}" with license "{license}" which this tool does not know about and '
+                      'therefore cannot give any recommendations.').format(plugin=entry_point.dist.key, license=license)
+                ))
+                continue
```

Comment:
Could avoid the `continue` and build an `elif` stack, which might also avoid multiple warnings for the same license, although that shouldn't happen.